docs/basics/rankine_cycle.rst

```diff
@@ -81,6 +81,40 @@ water return flow temperature.
 After rerunning, we will see that the condensation temperature and pressure
 are both automatically calculated by the specified terminal temperature value.
 
+Generating T-s Diagram
+^^^^^^^^^^^^^^^^^^^^^^
+To visualize the Rankine Cycle, we generate a temperature (T) versus 
+entropy (s) diagram using the fluprodia (Fluid Property Diagram) 
+package.
+
+.. dropdown:: Click to expand to code section
+
+    .. literalinclude:: /../tutorial/basics/rankine.py
+        :language: python
+        :start-after: [sec_Ts]
+        :end-before: [sec_5]
+
+The steps involved in generating the T-s diagram are as follows:
+- Import the Package: Import fluprodia and create an object by passing the alias of the fluid.
+- Specify the Unit System: Set the unit system for all fluid properties.
+- Specify Custom Isolines: Define custom isolines for the diagram.
+- Calculate and Draw Isolines: Calculate and draw the isolines based on the model results.
+- Save and Export the Diagram: Save and export the completed T-s diagram.
+
+.. figure:: /_static/images/basics/rankine_ts_diagram.svg
+    :align: center
+    :alt: T-s Diagram of Rankine Cycle
+    :figclass: only-light
+
+    Figure: T-s Diagram of Rankine Cycle
+
+Besides visualization, this feature is also useful for analysis purposes. 
+For example, if the T-s diagram forms a closed loop, validating the accuracy of the model and 
+that the operating fluid completes a successful Rankine Cycle. By applying fluprodia, 
+we can create and customize different types of diagrams for all pure and pseudo-pure fluids 
+available in CoolProp. For more information on fluprodia, we refer users to the
+`fluprodia documentation <https://fluprodia.readthedocs.io/en/latest/>`_.
```

Comment:
What is the ending underscore `_` about? I don't see that it is required for hyperlink

Reply:
Yes, I think should be a typo. Thanks for pointing out, I will make the changes.